app/views/user_mentee_application_cohorts/show.html.erb

```diff
@@ -6,7 +6,8 @@
   <div class="tabs">
     <%= link_to 'In Review', user_mentee_application_cohort_path(@cohort, filter: "in_review" ) , class:"tab tab-lifted #{'tab-active' if params[:filter] == 'in_review' || params[:filter].nil? } " %>
     <%= link_to 'Accepted', user_mentee_application_cohort_path(@cohort, filter: "accepted") , class:"tab tab-lifted #{'tab-active' if params[:filter] == 'accepted'}" %>
-    <%= link_to  "rejected", user_mentee_application_cohort_path(@cohort, filter: "rejected" ), class:"tab tab-lifted #{'tab-active' if params[:filter] == 'rejected' }" %>
+    <%= link_to  "Rejected", user_mentee_application_cohort_path(@cohort, filter: "rejected" ), class:"tab tab-lifted #{'tab-active' if params[:filter] == 'rejected' }" %>
+    <%= link_to  "Received", user_mentee_application_cohort_path(@cohort, filter: "application_received" ), class:"tab tab-lifted #{'tab-active' if params[:filter] == 'application_received' }" %>
```

Comment:
Can we have this be the default state and above in review. In the cases that there are no received applications (i.e: they are all in review, make that what is shown?)